onnx/defs/schema.h

```diff
@@ -988,7 +988,7 @@ OpSchema GetOpSchema();
 
 #define ONNX_PREVIEW_OPERATOR_SET_SCHEMA(name, ver, impl) \
```

Comment:
please also change this macro's name accordingly.

Reply:
ONNX_PREVIEW_TRAINING_OPERATOR_SET_SCHEMA look good?